recipes/runtimeqml/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,12 @@
+#include <QApplication>
+#include <QQmlApplicationEngine>
+
+#include <runtimeqml.hpp>
+
+int main(int argc, char* argv[]) {
+    QApplication app(argc, argv);
+    QQmlApplicationEngine engine;
+
+    RuntimeQml* rt = new RuntimeQml(&engine);
```

Comment:
New without a delete 🙈

Reply:
We don't run Valgrind here 😈

Reply:
Fixed. (Although in their documentation it does say to do this, Weird).

Reply:
Sorry c++ dev at heart